docs/homepage/blog/ospp_mid-term_report_210370741/index.md

```diff
@@ -0,0 +1,393 @@
+- [RL Unplugged: Benchmarks for Offline Reinforcement Learning](https://github.com/deepmind/deepmind-research/tree/master/rl_unplugged)
+
+Make standard policies in [Benchmarks for Deep Off-Policy Evaluation](https://github.com/google-research/deep_ope) to be available in RLDatasets.jl.
+
+Implement an **Off Policy Evaluation** method and select between a number of standard policies for a particular task using RLDatasets.jl.
+
+The following are the future work that are possible in this project.
+
+- Parallel loading and partial loading of datasets.
+- Add support for `environments` that are not supported by GymEnvs -> Flow and CARLA.
+- Add support for datasets in Flow and CARLA envs.
+- Add support for creating, storing and loading custom made datasets.
+- `test-train` split functionality for datasets.
+- Cross validation and grid search.
+- Enable features that make a particular algorithm based on the requirements of the env.
+- `evaluator` function that performs evaluation (can be on policy or off policy)
+- Metrics as hooks. Refer [Metrics](https://d3rlpy.readthedocs.io/en/v0.90/references/metrics.html)
+
+Refer the following [discussion](https://github.com/JuliaReinforcementLearning/ReinforcementLearning.jl/discussions/359) for more ideas.
+
+### Time Planning
+
+| Date       | Goals |
+| :-----------: | :---------: |
+| 07/01 - 07/14 | Brainstorm various ideas that are possible for the implementation of RLDatasets.jl and finalize the key features. |
+| 07/15 - 07/20 | Made a basic julia wrapper for `d4rl` environments and add some tests |
+| 07/21 - 07/30 | Implemented `d4rl` and `d4rl-pybullet` datasets |
+| 07/31 - 08/06 | Implemented  `Google Research DQN Replay Datasets` |
+| 08/07 - 08/14 | Implemented `RL Unplugged atari datasets`, setup the docs, added README.md. Made the package more user friendly. Make the **mid-term report** |
+| 08/15 - 08/30 | Add lazy loading and multi threaded loading support for `Google Research DQN Replay Datasets`. Add the rest of RL Unplugged datasets, polish the interface, finalize the structure of the codebase. Add examples and `register the package`.|
+| 09/01 - 09/15 | Add support for policy loading from [Benchmarks for Deep Off-Policy Evaluation](https://github.com/google-research/deep_ope) and implement an `Off Policy Evaluation` method |
+| 09/16 - 09/30 | Test OPE in various datasets and publish benchmarks in RLDatasets.jl. Implement other features that makes the package more user friendly. Complete the **final-term report** |
+
+## 3. Implemented datasets
+
+### D4RL
+
+Added support for D4RL datasets with all features loaded in the returned type.
+
+Credits: [D4RL](https://github.com/rail-berkeley/d4rl)
+
+```julia
+using ReinforcementLearningDatasets
+ds = dataset(
+        "hopper-medium-replay-v0";
+        repo="d4rl")
+```
+
+The type (`D4RLDataSet`) returned by `dataset` is an `Iterator` that returns batches of data based on the requirement that is specified.
```

Comment:
Given that we've setup the docs of the `RLDatasets` package, better to provide a link to it in https://juliareinforcementlearning.org/docs/rldatasets/